maths/shreedharacharya.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:math';
+//this function return a list of roots of a quadratic equation
+// [x1, x2] where x1 and x2 are roots of
+// aX^2 + bX + c = 0
+List shreedharacharya(double a,double b,double c) 
+{
+  double d = b*b - 4*a*c;
+  List A=[0,0];
```

Comment:
You can just create empty List and use `.add` method to extend it. List doesn't have fixed length anyway.
```suggestion
  List A= [];
```
It's important, because depending on `d` you can have 0, 1 or 2 real roots.

Reply:
Ok sir will do it